pnpm-lock.yaml

```diff
@@ -229,7 +229,7 @@ importers:
       '@prisma/debug': workspace:*
       '@prisma/engine-core': workspace:*
       '@prisma/engines': workspace:*
-      '@prisma/engines-version': 4.9.0-32.f307669e852dde0c751a36420b11f932fc3cf268
+      '@prisma/engines-version': 4.9.0-33.bb177fac1c08b8561c02b04d9ad51a31981cd8a3
```

Comment:
Diff in the pnpm lock is unexpected, did you install this engine before it landed in main?

Reply:
Yes, I had tried it out before it landed in main to see whether all tests passed. I can revert https://github.com/prisma/prisma/pull/16766/commits/fcbd3bcf9cf70db9125b12ea70b59739d2fc0243 and merge `main` back to this branch

Reply:
It's probably fine in the end but you will get a conflict in this PR once a new engine version lands

Reply:
I have already done the steps described in my previous message with no conflicts whatsoever :)